src/service/llm.rs

```diff
@@ -151,6 +184,12 @@ impl OpenAiLlmClient {
                     .content(format!("## Web Search Results\n\n{}\n\n", context.web_search_context))
                     .build()?,
             ),
+            InputItem::Message(
+                InputMessageArgs::default()
+                    .role(ResponsesRole::Developer)
+                    .content(format!("## Channel Message History\n\n{}\n\n", context.message_search_context))
```

Comment:
Can we make this "message search context" so the agent knows that a search of some kind was performed?